plugins/gpu/nvidia_gpu_

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Please stick with `/bin/bash` for now.
The interpreter detection of our tests currently fail on the above `env`. Thus for now `/bin/bash` is the way.
Please adjust locally, if necessary.